stories/forms/labels/input-label.story.js

```diff
@@ -35,6 +35,12 @@ storiesOf('InputLabel', module)
       requiredIndicator={ '*' }
     />
   ))
+  .add('pass DOM props to inner label tag', () => (
+    <InputLabel
+      name="nameOfInput"
+      aria-label="testAriaLabel"
```

Comment:
Can you swap this out with something like `data-cy`? Passing `aria-label` to a label element is not interpreted correctly by VoiceOver.